app/build.gradle

```diff
@@ -9,7 +9,7 @@ if (keystorePropertiesFile.exists()) {
 }
 
 android {
-    compileSdk = 34
+    compileSdk 33
```

Comment:
Is it required to drop down the SDK version to 33?
Are there any ways to keep SDK 34 and run it successfully on Android 13 devs?